docs/linux/reporting_kernel_bugs.drawio

```diff
@@ -0,0 +1,287 @@
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-88" value="" style="endArrow=none;dashed=1;html=1;rounded=0;exitX=0;exitY=0.5;exitDx=0;exitDy=0;startArrow=classic;startFill=1;" edge="1" parent="1" source="oGheDNism3NP_6yb4b1f-56">
+          <mxGeometry width="50" height="50" relative="1" as="geometry">
+            <mxPoint x="70" y="580" as="sourcePoint" />
+            <mxPoint x="280" y="390" as="targetPoint" />
+            <Array as="points">
+              <mxPoint x="200" y="621" />
+              <mxPoint x="200" y="390" />
+            </Array>
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-89" value="" style="endArrow=classic;html=1;rounded=0;exitX=0;exitY=0.5;exitDx=0;exitDy=0;fillColor=#f8cecc;strokeColor=#b85450;" edge="1" parent="1" source="oGheDNism3NP_6yb4b1f-72">
+          <mxGeometry width="50" height="50" relative="1" as="geometry">
+            <mxPoint x="10" y="630" as="sourcePoint" />
+            <mxPoint x="80" y="270" as="targetPoint" />
+            <Array as="points">
+              <mxPoint x="60" y="700" />
+              <mxPoint x="60" y="270" />
+            </Array>
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-90" value="" style="endArrow=none;html=1;rounded=0;fillColor=#f8cecc;strokeColor=#b85450;" edge="1" parent="1">
+          <mxGeometry width="50" height="50" relative="1" as="geometry">
+            <mxPoint x="80" y="690" as="sourcePoint" />
+            <mxPoint x="80" y="500" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-91" value="&lt;span style=&quot;font-size: 10px;&quot;&gt;Embargo can be requested by reporter to coordinate linux-distros, otherwise the fix is merged ASAP per security@kernel.org interest&lt;/span&gt;" style="rounded=0;whiteSpace=wrap;html=1;" vertex="1" parent="1">
+          <mxGeometry x="550" y="350" width="150" height="80" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-93" value="&lt;div&gt;&lt;span style=&quot;font-size: 10px;&quot;&gt;once the fix is merged to the stable tree, do NOT include security@kernel.org in discussion&lt;/span&gt;&lt;/div&gt;" style="rounded=0;whiteSpace=wrap;html=1;" vertex="1" parent="1">
+          <mxGeometry x="550" y="586.25" width="130" height="70" as="geometry" />
+        </mxCell>
+        <mxCell id="oGheDNism3NP_6yb4b1f-95" value="&lt;div&gt;&lt;span style=&quot;font-size: 10px;&quot;&gt;CVEs are assigned&lt;/span&gt;&lt;/div&gt;&lt;div&gt;&lt;span style=&quot;font-size: 10px;&quot;&gt;after-the-fact of the fix is merged by CVE assignment team (see linux-cve-announce list). If the applicable CVE is missed, or reporter needs CVE assigned before an issue is resolved with a commit, notify cve@kernel.org&lt;/span&gt;&lt;/div&gt;" style="rounded=0;whiteSpace=wrap;html=1;" vertex="1" parent="1">
```

Comment:
Why does the font change before "after-the-fact"?

Also, we can make it shorter:

CVEs are typically assigned after the fix is merged (see linux-cve-announce list). If a CVE is missing or needed before the fix reaches upstream, contact cve@kernel.org.